File: test_archives/cpp_functions/batch5/function0.cpp
#include <iostream>
#include <string>
#include <vector>
#include <map>
#include <set>
#include <tuple>

using namespace std;

class Function0 {
public:
    int integer;
    float float;
    string string;
    vector<int> list;
    map<char, int> dictionary;
    tuple<int, int, int> tuple;
    set<int> set;
    
    Function0(int int_input, float float_input, string string_input, vector<int> list_input) {
        integer = int_input;
        float = float_input;
        string = string_input;
        list = list_input;
        dictionary = {{'a', 1}, {'b', 2}, {'c', 3}};
        tuple = make_tuple(1, 2, 3);
        set = {1, 2, 3};
    }
    
    double multiply_result() {
        double result = integer * float;
        for (int i : list) {
            result *= i;
        }
        return result;
    }
};

File: test_archives/py_functions/batch5/function0.py
class function0:
    def __init__(self, int_input, float_input, string_input, list_input):
        self.integer = int_input
        self.float = float_input
        self.string = string_input
        self.list = list_input
        self.dictionary = {'a': 1, 'b': 2, 'c': 3}
        self.tuple = (1, 2, 3)
        self.set = {1, 2, 3}
 
    def multiply_result(self):
        result = self.integer * self.float
        for i in self.list:
            result *= i
        return result